bincache/Cargo.toml

```diff
@@ -6,17 +6,21 @@ description = "ZitaneLabs binary cache."
 
 [features]
 default = ["implicit-blocking"]
-implicit-blocking = []
-blocking = []
-rt_tokio_1 = ["dep:tokio"]
-rt_async-std_1 = ["dep:async-std"]
+implicit-blocking = ["dep:futures-io", "dep:futures-util", "async-compression/futures-io"]
+blocking = ["dep:futures-io", "dep:futures-util", "async-compression/futures-io"]
+rt_tokio_1 = ["dep:tokio", "async-compression/tokio"]
+rt_async-std_1 = ["dep:async-std", "dep:futures-io", "dep:futures-util", "async-compression/futures-io"]
 
 [dependencies]
 paste = "1.0.12"
 thiserror = "1.0.40"
 async-trait = { version = "0.1" }
 tokio = { version = "1", features = ["rt", "fs", "io-util"], optional = true }
 async-std = { version = "1", optional = true }
+zstd = "0.12.3"
```

Comment:
We should probably feature-gate this. Mabye a `comp_zstd` feature? So we can add more stuff later on (`comp_brotli`, `comp_gzip`, etc.) without growing the binary size.

Also, is the `zstd` dependency actually needed? Or does `async_compression` with the `zstd` feature include it? Maybe run `cargo +nightly udeps` to find out

Reply:
Zstd is not needed anymore as we use async-compression. Removed it. For feature gating we have to think about another possible solution in the future. It is not possible to make async-compression optional right now due to the other features.